locales/es-ES.json

```diff
@@ -436,20 +452,21 @@
   },
   "tooltip": {
     "add_content_warning": "Añadir advertencia de contenido",
+    "add_emojis": "Añadir emojis",
     "add_media": "Añadir imágenes, video o audio",
-    "add_publishable_content": "Agregar contenido a publicar",
+    "add_publishable_content": "Publicar contenido",
     "change_content_visibility": "Cambiar visibilidad de contenido",
     "change_language": "Cambiar idioma",
     "emoji": "Emoji",
-    "explore_links_intro": "Estas noticias están siendo comentadas por la gente en este y otros servidores de la red descentralizada en este momento.",
-    "explore_posts_intro": "Estos mensajes de este y otros servidores de la red descentralizada están ganando tracción en este servidor en este momento.",
-    "explore_tags_intro": "Estas etiquetas están ganando tracción entre la gente de este y otros servidores de la red descentralizada en este momento.",
+    "explore_links_intro": "Estas noticias están siendo comentadas por los usuarios de este y otros servidores de la red descentralizada ahora mismo.",
+    "explore_posts_intro": "Estos mensajes de este y otros servidores de la red descentralizada están siendo tendencia en este servidor ahora mismo.",
```

Comment:
¿Este mensaje es que puedes tener mensajes de distintos servidor mostrados en el servidor actual? No se si esto es posible. De todas formas yo pondría el `ahora mismo` después de `siendo tendencia`.

Reply:
La frase original es `These posts from this and other servers in the decentralized network are gaining traction on this server right now.` Y correcto, lo he puesto como tu sugerencia.

Reply:
te dejaste este `ahora mismo`: L462